Makefile

```diff
@@ -1,6 +1,11 @@
 all: lkmpg.tex
 	pdflatex -shell-escap lkmpg.tex
 
+html: 
+	make4ht -suf html5 -c html.cfg -d html lkmpg.tex
+
 clean:
-	rm -f *.dvi *.aux *.log *.ps *.pdf *.out lkmpg.bbl lkmpg.blg lkmpg.lof lkmpg.toc
-	rm -rf _minted-lkmpg
+	rm -f *.dvi *.aux *.log *.ps *.pdf *.out lkmpg.bbl lkmpg.blg lkmpg.lof lkmpg.toc *.xref *.svg *.tmp *.html *.css *.4ct *.4tc *.dvi *.lg *.idv
```

Comment:
Move HTML-specific files to the next line.

Reply:
done